api/users.rb

```diff
@@ -19,6 +19,14 @@
   end
 end
 
+get "#{APIPREFIX}/users/:user_id/content_exists" do |user_id|
+  begin
+    Content.where(author_id: user_id).exists?.to_json
+  rescue Mongoid::Errors::DocumentNotFound
+    error 404
```

Comment:
I don't know if this is possible to hit with the query I'm trying to run, but like, I figured it can't hurt.